lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -927,6 +927,7 @@ class SemanticsObject {
           ..height = '${rect.height}px';
       }
     } else {
+      // This condition creates a issue in scrolling.
```

Comment:
Please add more detail, what issue?

Reply:
ah, thanks for noticing. I was planning to do that. I even added this note to not forget.

Reply:
done.